lib/features/scan_nfc/widgets/android_nfc_found_bottomsheet.dart

```diff
@@ -1,18 +1,10 @@
 import 'package:flutter/material.dart';
 import 'package:flutter/widgets.dart';
 import 'package:flutter_svg/svg.dart';
-import 'package:givt_app_kids/features/scan_nfc/cubit/scan_nfc_cubit.dart';
 import 'package:givt_app_kids/shared/widgets/givt_elevated_button.dart';
 
 class FoundNfcAnimation extends StatelessWidget {
-  const FoundNfcAnimation(
-      {required this.scanNfcCubit,
-      required this.isLoading,
-      required this.onPressed,
-      super.key});
-  final ScanNfcCubit scanNfcCubit;
-  final bool isLoading;
-  final void Function()? onPressed;
+  const FoundNfcAnimation({super.key});
```

Comment:
These were not used in the widget, i think they were remainders from previous versions of this feature.